R/pk.calc.all.R

```diff
@@ -378,6 +379,11 @@ pk.nca.interval <- function(conc, time, volume, duration.conc,
     }
     conc <- impute_data$conc
     time <- impute_data$time
+    tmp_imp_method <- c(
+      paste0("Imputation: ", paste(na.omit(impute_method), collapse = ", "))
+    )
+  } else {
+    tmp_imp_method <- c()

```

Comment:
Please change this to `character()` instead of `c()` to be clearer about the intended class.